storage/remote/queue_manager.go

```diff
@@ -105,35 +112,39 @@ type StorageClient interface {
 	Name() string
 }
 
+// StorageQueueManagerConfig configures a storage queue.
 type StorageQueueManagerConfig struct {
```

Comment:
We could drop the `Storage` prefix here. It's the "remote storage" package so sufficient context should be established.